assets/javascript/game.js

```diff
@@ -4,42 +4,61 @@ var guessWrong = 0;
 var maxGuesses = 7;
 var winCount = 0;
 var lossCount = 0;
-var wordBank = ["Algebra", "Poodle", "Citizen", "Pizzaz", "Cooperate", "Munchkin", "Flower", "Supercalifragilisticexpialidocious"];
```

Comment:
so, my vscode is set up to automatically use single-quotes so it reformatted and replaced them all.